src/hdprivatekey.js

```diff
@@ -547,14 +547,6 @@ HDPrivateKey.prototype.toString = function() {
   return this.xprivkey;
 };
 
-/**
- * Returns the console representation of this extended private key.
- * @return string
- */
-HDPrivateKey.prototype.inspect = function() {
```

Comment:
Just curious what was the reason for this removal? Sure, `console.log()` in production code isn't the best thing ever, but this could stay I suppose?